sql-prepared-plan-cache.md

```diff
@@ -6,6 +6,13 @@ aliases: ['/tidb/dev/sql-prepare-plan-cache']
 
 # SQL Prepared Execution Plan Cache
 
+> **Warning:**
+>
+> If a cached `UPDATE` or `DELETE` statement encounters a DDL operation that modifies the relevant schema during execution, it might cause data inconsistency between tables and indexes. For more information, see [Issue #51407](https://github.com/pingcap/tidb/issues/51407). Keep updated to the progress of [the latest LTS version](https://docs.pingcap.com/tidb/stable) to resolve this issue. The following are possible workarounds before upgrading:
```

Comment:
```suggestion
> If a cached `UPDATE` or `DELETE` statement encounters a DDL operation that modifies the relevant schema during execution, it might cause data inconsistency between tables and indexes. For more information, see [Issue #51407](https://github.com/pingcap/tidb/issues/51407). It is recommended to monitor the status of this issue and upgrade to [the latest LTS version](https://docs.pingcap.com/tidb/stable) to resolve this issue. Before upgrading, you can try the following workarounds:
```